cowtown2023/src/main/java/com/team2357/frc2023/subsystems/IntakeRollerSubsystem.java

```diff
@@ -0,0 +1,59 @@
+package com.team2357.frc2023.subsystems;
+
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.team2357.frc2023.Constants;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class IntakeRollerSubsystem extends SubsystemBase {
+
+    private CANSparkMax m_topIntakeMotor;
+    private CANSparkMax m_bottomIntakeMotor;
+
+    public IntakeRollerSubsystem() {
+        m_topIntakeMotor = new CANSparkMax(Constants.CAN_ID.TOP_INTAKE_ROLLER_MOTOR_ID, MotorType.kBrushless);
+        m_bottomIntakeMotor = new CANSparkMax(Constants.CAN_ID.BOTTOM_INTAKE_ROLLER_MOTOR_ID, MotorType.kBrushless);
+    }
+
+    public void configure() {
```

Comment:
It does not look like you are calling this method. Also, I would be okay with getting rid of this method in our subsystems and moving the config to the constructor.

Reply:
I'm gonna keep it as a separate method for organization purposes